CMakeLists.txt

```diff
@@ -66,7 +66,9 @@ configure_file(
   "${PROJECT_SOURCE_DIR}/src/qcdloop.pc.in"
   "${PROJECT_SOURCE_DIR}/src/qcdloop.pc"
   )
-IF(${CMAKE_SYSTEM_PROCESSOR} MATCHES "x86_64")
+# Only enable quadmath for non-aarch64 as aarch64 supports float128 and so
+# doesn't have libquadmath
+IF(NOT ${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64")
```

Comment:
@henryiii is there a more preferred way to do this? Or is just checking for `aarch64` somewhere good enough?

Reply:
Hm. From https://github.com/conda-forge/qcdloop-feedstock/pull/7 it seems like maybe `${CMAKE_SYSTEM_PROCESSOR}` isn't what is desired when cross-compiling, as when cross-compiling for `aarch64` on `x86` it is attempting to link against `libquadmath`.